src/components/pages/NewParentDashboard/NewParentDashboard.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Logout from '../../common/Logout';
+import {Link} from 'react-router-dom';
+import { Layout, Button, PageHeader, Menu, Typography } from 'antd';
+
+// Styling still needs work on this page
+export default function NewParentDashboard(props) {
+    
+    return(
+        <Layout>
+            <PageHeader style={{color:"hotpink", fontSize:'5rem'}}>Parent Dashboard</PageHeader>
+            <Menu id="parentDB">
+                <div>  
+                    <Button>Child Progress/Dashboard</Button>
+                </div>
+                {/* Need to figure out how to put link in a onclick for the button */}
```

Comment:
What link are you looking to put in here?

Reply:
We figured that out, so I took that comment out

Reply:
I deleted that comment, we figured it out